linux/open_vm_tools/install_ovt.yml

```diff
@@ -52,3 +52,18 @@
 # Eject CDROM /dev/sr0 from guest
 - include_tasks: ../utils/eject_cdrom_in_guest.yml
```

Comment:
Is there a when condition missed here, "linux_ovt_repo_url is undefined or not linux_ovt_repo_url"?

Reply:
Fixed.